xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -65,10 +65,10 @@
         <Parameter Name="value" Type="System.Int32" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="reader">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="reader">The byte sequence reader instance from which the value is to be read.</param>
```

Comment:
nit: `The byte sequence reader instance` is a "mouthful" to read/parse.

Maybe we can reword it differently, or omit the T. Open to suggestions.

```text
The <see cref="SequenceReader" /> of byte from which to read the value.
```
OR
```text
The byte <see cref="SequenceReader" /> from which to read the value.
```

Reply:
A link to SequenceReader appears immediately above the parameter description, so it would be good to avoid the duplication. How about:

> The byte sequence reader from which to read the value.

Reply:
Works for me.

Reply:
Thanks both. I'll apply Ron's suggestion then:

```suggestion
        <param name="reader">The byte sequence reader from which to read the value.</param>
```